src/js/angular-socialshare.js

```diff
@@ -175,10 +175,26 @@
         };
 
         $scope.tumblrShare = function manageTumblrShare (data) {
-          //tumblr doesnt likes http:// or https:// actually its a mistery
-          $window.open(
-            '//www.tumblr.com/share/link?url=' + encodeURIComponent(data.url.replace('http://', '').replace('https://')) + '&description=' + encodeURI(data.text)
```

Comment:
this replace function is no longer needed, whatever bug was causing it has been fixed by the tumblr developers